api_data.py

```diff
@@ -10,15 +10,15 @@
 	from urllib import urlopen 
 
 def output_cve_id(type=None, name=None, version=None): 
-	"""Takes as input, a package name, package version. 
+	"""Take as input, a package name, package version. 
 	Queries cve-search' dataset for any reported 
 	vulnerabilities of the requested package. If 
 	vulnerability exists, outputs cve-id(s).
 	"""
 	if version:
-		url = 'https://cve.circl.lu/api/search/' + name + version
+		url = ('https://cve.circl.lu/api/search/{}/{}').format(name,version)
```

Comment:
Here the parenthesis around the string are not needed: `'{}'.format()`
Since we are using Python 3.6+, feel free to use the format string as in:
`f'https://cve.circl.lu/api/search/{name}/{version}'`